src/test/shell/tool/github/OssRulesOfPlayRatingMarkdownFormatter.config.yml

```diff
@@ -20,3 +20,5 @@ ruleIds:
   rl-security_policy-1: If an open-source project has a security policy
   rl-code_of_conduct_file-1: If a project has a code of conduct file
   rl-code_of_conduct_file-2: If a project's code of conduct guideline has required text
+  rl-codescan_alerts-1: If a project has code scan enabled
+  rl-codescan_alerts-2: If a project has code scan on pull requests enabled 
```

Comment:
Here `If a project runs CodeQL checks for commits`

Reply:
Changed it to "Does it run CodeQL scans?" and " Does it use CodeQL checks for pull requests? " to match the mention in other places in the code .

Reply:
That would not work. Here the rules are being mapped to a specific feature in OssFeatures.java. Please check there for CodeQL, as to what the mapping should be.